js/renovation/ui/scroll_view/scrollable_simulated.tsx

```diff
@@ -1189,10 +1193,10 @@ export class ScrollableSimulated extends JSXComponent<ScrollableSimulatedPropsTy
     }
 
     // T320141
-    const isOverflowHidden = getElementComputedStyle(this.contentRef.current)!.overflowY === 'hidden';
+    const computedStyle = getElementComputedStyle(this.contentRef.current);
 
     /* istanbul ignore next */
-    if (isOverflowHidden) {
+    if (computedStyle.overflowY === 'hidden') {
```

Comment:
Can we change this declaration:
```
export default function getElementComputedStyle(el: Element | undefined | null):
CSSStyleDeclaration | null {
```
to this:
```
export default function getElementComputedStyle(el: Element):
CSSStyleDeclaration {
```
?
Or, even use 'window.getComputedStyle.(el)' directly without wrappers?
It will resolve such monkey changes with "ignore no-null-assertion" comments.

Reply:
Let's extract this change to a new PR.